libvmi/core.c

```diff
@@ -565,13 +565,13 @@ vmi_init_custom(
     } else if (VMI_CONFIG_GHASHTABLE == config_mode) {
 
         char *name = NULL;
-        unsigned long domid = VMI_INVALID_DOMID;
+        uint32_t domid = VMI_INVALID_DOMID;
         GHashTable *configtbl = (GHashTable *)config;
         gpointer idptr = NULL;
 
         name = (char *)g_hash_table_lookup(configtbl, "name");
         if(g_hash_table_lookup_extended(configtbl, "domid", NULL, &idptr)) {
```

Comment:
I nominate for a PR in parallel with this one, Maybe dropped in notes/  (??)

In libvmi/core.c
<https://github.com/libvmi/libvmi/pull/255#discussion_r28254189>:

> @@ -565,13 +565,13 @@ vmi_init_custom(
>      } else if (VMI_CONFIG_GHASHTABLE == config_mode) {
>
>          char *name = NULL;
> -        unsigned long domid = VMI_INVALID_DOMID;
> +        uint32_t domid = VMI_INVALID_DOMID;
>          GHashTable *configtbl = (GHashTable *)config;
>          gpointer idptr = NULL;
>
>          name = (char *)g_hash_table_lookup(configtbl, "name");
>          if(g_hash_table_lookup_extended(configtbl, "domid", NULL, &idptr)) {

Not really at the moment..

—
Reply to this email directly or view it on GitHub
<https://github.com/libvmi/libvmi/pull/255/files#r28254189>.